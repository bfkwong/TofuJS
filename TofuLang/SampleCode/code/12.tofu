print(toNumber("3"));
print(toString(4));
print(power(2, 2));
print(sqrt(4, 2));
print(strsplit("Hello World", " "));
print(strjoin(["Hello", "World"], " "));
print([1,2,3] + [4,5]);