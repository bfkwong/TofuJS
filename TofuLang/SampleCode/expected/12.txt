3
4
4
2
[ 'Hello', 'World' ]
Hello World
[ 1, 2, 3, 4, 5 ]
